lib/hopper.rb

```diff
@@ -100,20 +108,42 @@ def semaphore
       @semaphore ||= Mutex.new
     end
 
+    def connection
+      thread_local_variable(:hopper_connection) do
+        options = Hopper::Configuration.configuration
```

Comment:
Keeping the list open of configs so we don't have to change the library every time we want to use a new option. Ie I want to add heartbeat=0 to disable heartbeats as recommended by CloudAMQP